servo-gecko-try/files/hgrc

```diff
@@ -0,0 +1,7 @@
+[extensions]
+mq =
+push-to-try = ~/.mozbuild/version-control-tools/hgext/push-to-try
+
+[ui]
+username = Bors <whatever@whatever.com>
+ssh = ssh -l 'borsusername@bors.com' -i '/home/servo/.ssh/id_rsa.pub'
```

Comment:
Can we update the identity to match `bors-servo`?
Also update the key location.